altair_widgets/widget.py

```diff
@@ -1,9 +1,26 @@
+from subprocess import call
+
+call("jupyter nbextension enable --py --sys-prefix widgetsnbextension", shell=True)
+call("jupyter nbextension enable --py --sys-prefix vega", shell=True)
+call("pip install altair_saver", shell=True)
+call("pip install jupyter pandas vega", shell=True)
+call("pip install --upgrade notebook", shell=True)
+call("jupyter nbextension install --sys-prefix --py vega", shell=True)
+call("apt-get update", shell=True)
+call("apt-get -qq install chromium-chromedriver", shell=True)
```

Comment:
I think a better place for this code would be in the README (this code shouldn't run everytime someone types `import altair_widgets`, and this code may not work for everyone not everyone (e.g., if they don't have the `apt-get` command)).